docs/src/pages/guides/interoperability/interoperability.md

```diff
@@ -599,3 +599,138 @@ It works exactly like styled components. You can [use the same guide](/guides/in
 ### The `styled()` API
 
 It works exactly like styled components. You can [use the same guide](/guides/interoperability/#styled-components).
+
+## Tailwind CSS
+
+![stars](https://img.shields.io/github/stars/tailwindlabs/tailwindcss.svg?style=social&label=Star)
+![npm](https://img.shields.io/npm/dm/tailwindcss)
+
+If you are used to Tailwind CSS and want to use it together with the MUI components, you can start by cloning the [Tailwind CSS](https://github.com/mui-org/material-ui/tree/master/examples/tailwind) example project.
+If you use a different framework, or already have set up your project, follow these steps:
+
+1. Add Tailwind CSS to your project, following the instructions in https://tailwindcss.com/docs/installation.
+2. Remove the Tailwind's `base` directive in favor of the `CSSBaseline` component provided by `@mui/material`, as it plays nicer with the MUI components.
+
+**index.css**
+
+```diff
+-@tailwind base;
+ @tailwind components;
+ @tailwind utilities;
+```
+
+3. Add the `important` option, using the id of your app wrapper, for example `"#root"`.
+
+**tailwind.config.js**
+
+```diff
+ module.exports = {
+   content: [
+     "./src/**/*.{js,jsx,ts,tsx}",
+   ],
++  important: '#root',
+   theme: {
+     extend: {},
+   },
+   plugins: [],
+ }
+
+```
+
+This is necessary for ensuring that the [deeper elements](#deeper-elements-5), can be customized using the utility Tailwind's classes.
+More details on this option can be found here https://tailwindcss.com/docs/configuration#selector-strategy
+
+4. Fix the CSS injection order
+
+**Note:** Most CSS-in-JS solutions inject their styles at the bottom of the HTML `<head>`, which gives MUI precedence over your custom styles. To remove the need for **!important**, you need to change the CSS injection order. Here's a demo of how it can be done in MUI:
+
+```jsx
+import * as React from 'react';
+import { StyledEngineProvider } from '@mui/material/styles';
+
+export default function GlobalCssPriority() {
+  return (
+    <StyledEngineProvider injectFirst>
+      {/* Your component tree. Now you can override MUI's styles. */}
+    </StyledEngineProvider>
+  );
+}
+```
+
+**Note:** If you are using emotion and have a custom cache in your app, that one will override the one coming from MUI. In order for the injection order to still be correct, you need to add the prepend option. Here is an example:
```

Comment:
```suggestion
**Note:** If you are using emotion and have a custom cache in your app, it will override the one coming from MUI. In order for the injection order to still be correct, you need to add the prepend option. Here is an example:
```